WcaOnRails/lib/auxiliary_data_computation.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+module AuxiliaryDataComputation
+  def self.compute_everything
+    self.compute_best_of_3_in_333bf
+    self.compute_concise_results
+    self.compute_rank_tables
+
+    self.delete_php_cache # Note: this should go away together with the PHP code.
```

Comment:
I don't get how could this be a DB migration. This should be done each single time we run the script.